modules/service/src/test/scala/lucuma/odb/graphql/query/observationValidations.scala

```diff
@@ -1,690 +0,0 @@
-// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
```

Comment:
The `validations` field was removed and is now on `workflow.validationErrors`. The associated tests (in `observation_workflow`) include the tests here.